src/main/resources/db/migration/data/R__3_7__OFFENDER_IDENTIFIERS.sql

```diff
@@ -15,3 +15,7 @@ INSERT INTO OFFENDER_IDENTIFIERS (OFFENDER_ID, OFFENDER_ID_SEQ, IDENTIFIER_TYPE,
 INSERT INTO OFFENDER_IDENTIFIERS (OFFENDER_ID, OFFENDER_ID_SEQ, IDENTIFIER_TYPE, IDENTIFIER, ISSUED_DATE, ROOT_OFFENDER_ID, CASELOAD_TYPE, VERIFIED_FLAG ) VALUES (-1041, 2, 'CRO', '  01/11x  ', TO_DATE('2017-08-28', 'YYYY-MM-DD'), -1040, 'INST', 'Y');
 INSERT INTO OFFENDER_IDENTIFIERS (OFFENDER_ID, OFFENDER_ID_SEQ, IDENTIFIER_TYPE, IDENTIFIER, ISSUED_DATE, ROOT_OFFENDER_ID, CASELOAD_TYPE, VERIFIED_FLAG ) VALUES (-1042, 2, 'CRO', '  Sf99/12345m ', TO_DATE('2017-08-28', 'YYYY-MM-DD'), -1040, 'INST', 'Y');
 INSERT INTO OFFENDER_IDENTIFIERS (OFFENDER_ID, OFFENDER_ID_SEQ, IDENTIFIER_TYPE, IDENTIFIER, ISSUED_DATE, ROOT_OFFENDER_ID, CASELOAD_TYPE, VERIFIED_FLAG ) VALUES (-1003, 3, 'PNC', '24/1234567L', TO_DATE('2024-01-01', 'YYYY-MM-DD'), -1003, 'INST', 'Y');
+INSERT INTO OFFENDER_IDENTIFIERS (OFFENDER_ID, OFFENDER_ID_SEQ, IDENTIFIER_TYPE, IDENTIFIER, ISSUED_DATE, ROOT_OFFENDER_ID, CASELOAD_TYPE, VERIFIED_FLAG ) VALUES (-1012, 1, 'PNC', '2024/1234588L', TO_DATE('2024-01-01', 'YYYY-MM-DD'), -1012, 'INST', 'Y');
+INSERT INTO OFFENDER_IDENTIFIERS (OFFENDER_ID, OFFENDER_ID_SEQ, IDENTIFIER_TYPE, IDENTIFIER, ISSUED_DATE, ROOT_OFFENDER_ID, CASELOAD_TYPE, VERIFIED_FLAG ) VALUES (-1012, 2, 'NINO', 'DR784343E', TO_DATE('2024-01-01', 'YYYY-MM-DD'), -1012, 'INST', 'Y');
+INSERT INTO OFFENDER_IDENTIFIERS (OFFENDER_ID, OFFENDER_ID_SEQ, IDENTIFIER_TYPE, IDENTIFIER, ISSUED_DATE, ROOT_OFFENDER_ID, CASELOAD_TYPE, VERIFIED_FLAG ) VALUES (-1013, 1, 'PNC', '24/1234588L', TO_DATE('2024-01-01', 'YYYY-MM-DD'), -1012, 'INST', 'Y');
```

Comment:
this is slightly confusing as the two PNCs are identical - one in short form and the other in long form.  Might be better to have different PNC values to be clearer what the intent is

Reply:
It's just one of the scenarios I've seen in real world. Key thing is that all identifiers for all aliases are returned